master/internal/resourcemanagers/fair_share.go

```diff
@@ -50,11 +50,41 @@ func (g groupState) String() string {
 }
 
 func (f *fairShare) Schedule(rp *ResourcePool) ([]*sproto.AllocateRequest, []*actor.Ref) {
+	defer f.updateJobs(rp) // CHECK does this work how I think it does?
```

Comment:
are you trying to send updates after the scheduling occurs? Then yes it should work :)

Reply:
yep exactly and I didn't want to store it in a temp var explicitly.

Reply:
thanks